app/gui/config.yaml

```diff
@@ -15,6 +15,10 @@ windowAppScopeThemeName: "theme"
 # This MUST be kept in sync with the corresponding value in `app/gui/src/constants.rs`.
 projectManagerEndpoint: "ws://127.0.0.1:30535"
 
+# The URL to the base path of the HTTP endpoints of the Project Manager.
+# This MUST be kept in sync with the corresponding value in `app/gui/src/constants.rs`.
```

Comment:
I don't see any `projectManagerHttpEndpoint` constant in `constants.rs.

Reply:
good catch, thanks. yeah, i just copied the docs from the entry above